### ISSUE TITLE
move extend and merge to cytoplasm

### ISSUE THREAD

**Issue by mousemke** [sociomantic/microbe#74]
trying to move further towards modularity, extend and merge are in both core and root.  (which is bad).  there was the same with toArray and toString, but it was much easier to rectify.  extend and merge are both complex functions.  It could also be argued that extend and merge are important to the function of the selector engine.

**Comment by mousemke** [comment]
this is way more complicated than it seems.  extend uses a lot of root functions, so that would mean moving them over.

on further review i actually suggest abandoning the modularity project. throwing blocked on this for now, opening a new issue.

**Comment by jason-braun-sociomantic-mouse** [comment]
closing this going ahead with #76
